backends/tofino/bf-p4c/common/extract_maupipe.cpp

```diff
@@ -1728,7 +1726,7 @@ class GetBackendTables : public MauInspector {
                 decl, mce->arguments,
                 // if this is a @hidden table it was probably created from statements in
                 // the apply, so include that context when looking for @in_hash annotations
-                table->getAnnotations()->getSingle("hidden"_cs) ? getContext() : nullptr);
+                table->getAnnotation("hidden"_cs) ? getContext() : nullptr);
```

Comment:
I think I will make a pass at some point to replace all these canonical names ("hidden", "name", "default" etc) with their representative annotation member.

Reply:
Yes. I have not changed this in the backend (though tried to change in frontend / midend)